tasks/README.md

```diff
@@ -271,6 +271,8 @@ Name | Summary | Category
 `task274_overruling_legal_classification`	| Given a sentence, classify it into overruling or non-overruling. | Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect. | Text Modification
 `task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task287_casehold_legal_incorrect_answer_generation` | Given a prompt from a judicial decision and multiple potential holdings, choose one of the incorrect options | Legal , Incorrect Answer Generation
+`task290_TELLMYWHY_question_answerability` | Given a story and a question, decide whether or not the question is answerable | Classification
```

Comment:
Why are we adding these new lines?

Reply:
> Why are we adding these new lines?

I didn't add those lines myself. My assumption is maybe those lines and tasks weren't present during the time I put pull request and they got added later, so when I was resolving the conflict to make it merge with the master branch it may have to get added because of that. That's my assumption. But I didn't add it myself.

Reply:
They were probably leaked when resolving conflicts. Dropped the lines.